CodeEdit/Features/Tasks/Models/CEActiveTask.swift

```diff
@@ -23,6 +27,10 @@ class CEActiveTask: ObservableObject, Identifiable, Hashable {
     var outputPipe: Pipe?
 
     private var cancellables = Set<AnyCancellable>()
+    private let statusSubject = PassthroughSubject<CETaskStatus, Never>()
+    var statusPublisher: AnyPublisher<CETaskStatus, Never> {
+        statusSubject.eraseToAnyPublisher()
+    }
```

Comment:
Where are `statusSubject` or `statusPublisher` used? I couldn’t find any usage of them

Reply:
Removed